utils/vscode/language-configuration.json

```
@@ -3,22 +3,30 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 {
-  "comments": {
-    "lineComment": "//"
-  },
-  "brackets": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ],
-  "autoClosingPairs": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ],
-  "surroundingPairs": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ]
+    "comments": {
+        // symbol used for single line comment. Remove this entry if your language does not support line comments
```

Per our general style guidance, we would like comments to start with a capital letter and end with a period, and for lines to be wrapped to 80 columns.
```suggestion
        // Symbol used for single line comment.
```